sql/updates/world/3.3.5/2021_08_19_03_world.sql

```diff
@@ -0,0 +1,6 @@
+DELETE FROM `smart_scripts` WHERE `entryorguid`=27893 AND `source_type`=0;
+UPDATE `creature_template` SET `speed_walk`=2.5, `speed_run`=2.5, `unit_flags` = 33554434, `flags_extra`=0,`AIName`='', `ScriptName`='npc_pet_dk_rune_weapon' WHERE `entry`=27893;
```

Comment:
Please, I removed UNIT_FLAG_NON_ATTACKABLE from 100 creatures not to add it again
And speed is too fine

Reply:
values sniffed, will remove the flag and add it on the script, I guess

Reply:
That flag is added only when creature starts a specific spell and is removed when creature finishes cast. It should not be added in DB nor manually in scripts
Doubt I just picked the wrong sniff

<details>

[1] UpdateType: CreateObject2
[1] GUID: Full: 0xF130006CF5022B28 Type: Creature Entry: 27893 Low: 142120
[1] Object Type: 3 (Unit)
[1] Update Flags: 100 (AttackingTarget, Living, StationaryObject)
[1] Movement Flags: 134217729 (Forward, SplineEnabled)
[1] Extra Movement Flags: 0 (None)
[1] Time: 2591103920
[1] Position: X: 187.86685 Y: 213.55557 Z: 40.899273
[1] Orientation: 1.908361315727233886
[1] Fall Time: 0
[1] Walk Speed: 2.5
[1] Run Speed: 8
[1] RunBack Speed: 4.5
[1] Swim Speed: 4.722222328186035156
[1] SwimBack Speed: 2.5
[1] Turn Speed: 7
[1] Fly Speed: 4.5
[1] FlyBack Speed: 3.141592741012573242
[1] Pitch Speed: 3.141592741012573242
[1] Spline Flags: 69632 (WalkMode, FinalTarget)
[1] Final Spline Target GUID: Full: 0xF130005D91020960 Type: Creature Entry: 23953 Low: 133472
[1] Spline Time: 1
[1] Spline Full Time: 534
[1] Spline ID: 1833088
[1] Spline Duration Multiplier: 1
[1] Spline Duration Multiplier Next: 1
[1] Spline Vertical Acceleration: 0
[1] Spline Start Time: 0
[1] [0] Spline Waypoint: X: 187.34132 Y: 212.7048 Z: 40.81594
[1] [1] Spline Waypoint: X: 187.86685 Y: 213.55557 Z: 40.81594
[1] [2] Spline Waypoint: X: 186.7802 Y: 216.65146 Z: 40.816097
[1] [3] Spline Waypoint: X: 186.45676 Y: 217.5729 Z: 40.81559
[1] [4] Spline Waypoint: X: 186.45676 Y: 217.5729 Z: 40.81559
[1] Spline Mode: 0 (Linear)
[1] Spline Endpoint: X: 186.45676 Y: 217.5729 Z: 40.81559
[1] Target GUID: Full: 0xF130005D91020960 Type: Creature Entry: 23953 Low: 133472
[1] OBJECT_FIELD_GUID: Full: 0xF130006CF5022B28 Type: Creature Entry: 27893 Low: 142120
[1] OBJECT_FIELD_TYPE: 9
[1] OBJECT_FIELD_ENTRY: 27893
[1] OBJECT_FIELD_SCALE_X: 1
[1] UNIT_FIELD_CREATEDBY: Full: x Type: Player Low: x Name: x
[1] UNIT_FIELD_TARGET: Full: 0xF130005D91020960 Type: Creature Entry: 23953 Low: 133472
[1] UNIT_FIELD_BYTES_0: 0/1/2/1
[1] UNIT_FIELD_HEALTH: 6326
[1] UNIT_FIELD_POWER3: 100/1.4E-43
[1] UNIT_FIELD_POWER4: 100/1.4E-43
[1] UNIT_FIELD_POWER5: 1000000/1.401298E-39
[1] UNIT_FIELD_MAXHEALTH: 6326/8.865E-42
[1] UNIT_FIELD_MAXPOWER3: 100/1.4E-43
[1] UNIT_FIELD_MAXPOWER4: 100/1.4E-43
[1] UNIT_FIELD_MAXPOWER5: 1000000/1.401298E-39
[1] UNIT_FIELD_LEVEL: 67/9.4E-44
[1] UNIT_FIELD_FACTIONTEMPLATE: 115/1.61E-43 (115)
[1] UNIT_VIRTUAL_ITEM_SLOT_ID: 15256/2.1378E-41
[1] UNIT_FIELD_FLAGS: 34113544
[1] UNIT_FIELD_FLAGS_2: 2080
[1] UNIT_FIELD_BASEATTACKTIME: 3500/4.905E-42
[1] UNIT_FIELD_UNK63: 3500/4.905E-42
[1] UNIT_FIELD_BOUNDINGRADIUS: 0.5
[1] UNIT_FIELD_COMBATREACH: 1
[1] UNIT_FIELD_DISPLAYID: 11686/1.6376E-41
[1] UNIT_FIELD_NATIVEDISPLAYID: 11686/1.6376E-41
[1] UNIT_MOD_CAST_SPEED: 1065353216/1
[1] UNIT_CREATED_BY_SPELL: 49028/6.8703E-41
[1] UNIT_FIELD_BYTES_2: 1/0/0/0
[1] UNIT_FIELD_HOVERHEIGHT: 1

</details>